jena-arq/Grammar/master.jj

```diff
@@ -99,7 +99,42 @@ import org.apache.jena.sparql.core.Quad ;
 #endif
 
 public class CLASS extends PARSERBASE
-{}
+{
+    boolean allowAggregatesInExpressions = false ;
```

Comment:
This is not in master.jj nowadays. Is this from the early forking?

Reply:
Most likely. Will remove tonight.